compiler/vm.nim

```diff
@@ -525,6 +525,10 @@ proc rawExecute(c: PCtx, start: int, tos: PStackFrame): TFullReg =
   var savedFrame: PStackFrame
   var regs: seq[TFullReg] # alias to tos.slots for performance
   move(regs, tos.slots)
+  defer:
```

Comment:
Is this the right way to do this? I could remove the `pc` and `tos` variables and use `c.pc` and `c.tos` directly everywhere instead.

Reply:
That would be worse for performance, don't use ``c.pc`` everywhere.